cfgov/unprocessed/apps/youth-employment-success/js/util.js

```diff
@@ -221,11 +221,23 @@ function toPrecision( str = '', precision = 0 ) {
   return String( ( Math.round( ( num * 1000 ) / 10 ) / 100 ).toFixed( precision ) );
 }
 
+function formatNegative( num ) {
+  let formattedTotal = num;
+
+  if ( num < 0 ) {
+    formattedTotal = `&#8722;${ Math.abs( num ) }`;
```

Comment:
Why is the use of an HTML code necessary here?

Reply:
The designer wanted the HTML entity vs the '-' sign, as the entity is slightly larger.

Reply:
Perhaps they want an m-dash '–' (option + -) versus minus '-'?
Anyway, I suppose the entity is fine.

Reply:
I feel like an emdash `—` is a little too big...I can ask the designer and see what they say!

Reply:
@anselmbradford The designer does prefer the look of the entity vs the emdash